deploy/clowdapp.yaml

```diff
@@ -0,0 +1,174 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: insights-results-aggregator
+objects:
+
+- kind: HorizontalPodAutoscaler
+  apiVersion: autoscaling/v1
+  metadata:
+    labels:
+      app: insights-results-aggregator
+    name: insights-results-aggregator
+  spec:
+    minReplicas: ${{MIN_REPLICAS}}
+    maxReplicas: ${{MAX_REPLICAS}}
+    scaleTargetRef:
+      apiVersion: v1
+      kind: Deployment
+      name: insights-results-aggregator
+    targetCPUUtilizationPercentage: 50
```

Comment:
This is lower than what we have in current deployment (80% - see https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/ccx-data-pipeline/insights-results-aggregator.yml#L213).

Not sure if it will matter or not, but it might =)